schemas/namespace/identity-namespace.example1.json

```diff
@@ -0,0 +1,7 @@
+{
+  "xdm:namespaceCode": "ECID",
+  "id": "4657",
```

Comment:
all of the values need to be properly namespaced

Reply:
ack

Reply:
Welcome to XDM